src/Common/Logging/SystemLogger.php

```diff
@@ -25,6 +25,9 @@ class SystemLogger implements LoggerInterface
 
     public function __construct()
     {
+        // default to debug, otherwise it's undefined and it stops logging.
+        $logLevel = Logger::DEBUG;
```

Comment:
this happens when this class is called prior to globals being brought in. this is why i migrated it away from a singleton (then was stuck with the setting even after globals are brought in. below this a placed a section to manually override via comment/uncomment  $logLevel when need debugging prior to globals.

Reply:
will need to remove above 2 lines